cse/indexing/InvertedIndexWriter.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, filepath):
 
     def __shouldDeltaMerge(self):
         # check memory usage
-        if self.__dIndex.estimatedSize() > (50 * 1024*1024):
+        if self.__dIndex.estimatedSize() > (500 * 1024*1024):
```

Comment:
magic number?

Reply:
yes :D based on some manual tests, should I create a new constant?

Reply:
Since it is used in more than 1 place, might be better